airflow/providers/google/cloud/_internal_client/secret_manager_client.py

```diff
@@ -75,7 +75,7 @@ def get_secret(self, secret_id: str, project_id: str, secret_version: str = "lat
             value = response.payload.data.decode("UTF-8")
             return value
         except NotFound:
-            self.log.error("Google Cloud API Call Error (NotFound): Secret ID %s not found.", secret_id)
+            self.log.info("Google Cloud API Call Error (NotFound): Secret ID %s not found.", secret_id)
```

Comment:
I would be in favor of even making it debug like AWS's secrets backend:

https://github.com/apache/airflow/blob/1486afdc9f3855e80f475dcaa2adc729faf3bd08/airflow/providers/amazon/aws/secrets/systems_manager.py#L183

Reply:
make sense, thanks